lib/tasks/scrape_gov_trails.rake

```diff
@@ -1,7 +1,7 @@
 namespace :scrape_gov_trails do 
   desc "Scrape trail's info from http://www.lcsd.gov.hk/en/healthy/hiking/"
 
-  task :get_trails => :trails do
+  task :get_trails => :environment do
```

Comment:
This is not done. that was just a skeleton. but we might not need it at all

On Mon, Feb 23, 2015 at 4:15 PM, Harry Chen <notifications@github.com>
wrote:

> In lib/tasks/scrape_gov_trails.rake
> <https://github.com/cchchoi1986/hikehk/pull/29#discussion_r25148484>:
>
> > @@ -1,7 +1,7 @@
> >  namespace :scrape_gov_trails do
> >    desc "Scrape trail's info from http://www.lcsd.gov.hk/en/healthy/hiking/"
> >
> > -  task :get_trails => :trails do
> > +  task :get_trails => :environment do
>
> This is not done right? Why are we merging this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/29/files#r25148484>.
>